community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/ParallelBatchImporter.java

```diff
@@ -374,7 +374,7 @@ private void importRelationships( NodeRelationshipCache nodeRelationshipCache,
 
     private static Configuration configWithRecordsPerPageBasedBatchSize( Configuration source, RecordStore<?> store )
     {
-        return Configuration.withBatchSize( source, store.getRecordsPerPage() * 100 );
+        return Configuration.withBatchSize( source, store.getRecordsPerPage() * 10 );
```

Comment:
It's always painful introducing a config option for something that should just work. I don't think there's any benefit with the 100x multiplier... measuring as we speak